avm/res/network/virtual-network-gateway/tests/e2e/bgpAPIPA/main.test.bicep

```
@@ -0,0 +1,88 @@
+targetScope = 'subscription'
+
+metadata name = 'VPN'
```

As mentioned, let's pick here a 
```suggestion
metadata name = 'VPN with APIPA BGP settings'
```